### ISSUE TITLE
Does it make sense to use it with reacstrap?

### ISSUE THREAD

**Issue by hanoii** [KyleAMathews/typography.js#198]
Is it OK to use both libraries or one would conflict with the other? reactrap needs the bootstrap css to work, which I take brings in fonts and stylings anyway.

**Comment by fullofcaffeine** [comment]
+1 I'm trying to use it with boostrap. For the most part, it works, but I have to do a lot of style hacking in order to prevent boostrap from overriding the styles from typography.js. See: https://github.com/gatsbyjs/gatsby/commit/5caf95e3029096ef722e676902c08813fc5176e7#r37034290.

**Comment by fullofcaffeine** [comment]
Hi, could any of the core members chime in to give some insights regarding this issue? I'm currently using Bootstrap 4 + Typopgrahy.js (after going through a somewhat long process solve a related issue ) and while it works, there are clearly some small issues, mainly related to Bootstrap overwritting Typography.js rules or vice-versa. At the moment, I'm deliberately loading Typography.js __after__ Bootstrap so that the fonts are actually set. 

I think the main rationale for using both is twofold: 1) Bootstrap is popular and well-known, it's a really productive framework after you get the hang of it (my case) 2) Typography.js is really nice typeface-wise and the rules and themes save up a lot of time. Doing the same manually on Bootstrap (choosing fonts, overwriting typeface rules etc) would be very tedious. So I think there's a huge potential here of using both together, but there's no mention of it anywhere in the docs :(

**Comment by fullofcaffeine** [comment]
This issue has been opened for more than an year. I'm not expecting immediate answers from open-source maintainers - I'm one myself and I know how it is - but it's a long time for not having a reply. Is this project still being maintained or is this even the right forum for questions?

**Comment by vincentreynaud** [comment]
@fullofcaffeine agreed! I am struggling to find the right setup to avoid `bootstrap` overwriting `typography.js`.

**Comment by fullofcaffeine** [comment]
@vincentreynaud Yeah, that's unfortunate. I managed to find a workaround but it was an ugly one. I don't recall exactly what it was now, but I'll try to refresh my memory soon and document here.
